src/conic_algorithm.jl

```diff
@@ -251,8 +251,11 @@ function MathProgBase.loadproblem!(m::PajaritoConicModel, c, A, b, cone_con, con
             error("Some indices in a constraint cone do not correspond to indices of vector b\n")
         end
 
-        inds_con[inds] += 1
+        for i in inds
```

Comment:
`inds_con[inds] .+= 1` should work

Reply:
For some reason this variant resulted in `ErrorException("Some indices in vector c do not correspond to indices of a variable cone\n")` in Julia v0.7 and v1.0.